csrc/kernel_ir.cpp

```diff
@@ -149,7 +149,7 @@ Allocate::Allocate(
     NVF_ERROR(buffer->isA<TensorView>());
     NVF_ERROR(buffer->as<TensorView>()->getMemoryType() == memory_type);
     const auto domain = buffer->as<TensorView>()->domain();
-    for (auto axis : domain->noReductions()) {
+    for (auto axis : TensorDomain::noReductions(domain->maybeAllocation())) {
```

Comment:
This was done to work around the error described in https://github.com/NVIDIA/Fuser/pull/1456/commits/910ff09862387b3783af1837c82d5d600dc8b893

Reply:
Can you give more details? Does `T5` in the example in the commit have an allocation domain?